hoot-rnd/src/main/cpp/hoot/rnd/schema/ImplicitTagRulesDatabaseDeriver.cpp

```diff
@@ -166,7 +170,7 @@ void ImplicitTagRulesDatabaseDeriver::_removeKvpsBelowOccurrenceThreshold(const
   LOG_DEBUG(cmd);
   if (std::system(cmd.toStdString().c_str()) != 0)
   {
-    throw HootException("Unable to sort input file.");
+    throw HootException("Unable to clean input file.");
```

Comment:
This is a cleaning operation and not a sorting operation, fixed the copy/paste issue.